src/classes/Accordion.js

```diff
@@ -90,10 +90,15 @@ export class Accordion
     loaded() {
         if (window.location.hash) {
             // If there is a URL hash, and the element concerned is an accordion, then activate it
-            if (document.querySelector(window.location.hash).classList.contains('accordion')) {
-                this.activate(document.querySelector(window.location.hash));
+            const el = document.querySelector(window.location.hash);
+
+            if (el && el.classList.contains('accordion')) {
+                this.activate(el);
+                return;
             }
-        } else if (this.options.openOnLoad) {
+        }
+
+        if (this.options.openOnLoad) {
```

Comment:
@ImClarky I think this still needs to be `else if`, otherwise we could get two activations happening at once (for the hash, and the first element with openOnLoad)

Reply:
I don't think it should... the `return;` should handle that no...?

Also if we revert to having an `else if` and we have a hash that is not related to the Accordion, say an anchor to a  title of a different section, then the `openOnLoad` wouldn't happen..?